lib/dynflow/dispatcher/client_dispatcher.rb

```diff
@@ -26,16 +26,95 @@ def success!(resolve_to)
         end
       end
 
+      # Class used for reducing the number of sent Pings among worlds.
+      # World's coordinator record include the time when was the world
+      # seen for the last time. This class can be used to query this
+      # information and determine whether the record is "fresh enough"
+      # or whether the Ping really needs to be sent.
+      class PingCache
+        # Format string used for formating and parsing times
+        TIME_FORMAT = '%Y-%m-%d %H:%M:%S.%L'.freeze
+
+        # Maximum age of a record in seconds, older records are not considered fresh
+        PING_CACHE_AGE = 60
```

Comment:
I wonder if we shouldn't expose this and the `HEARTBEAT_INTERVAL` via dynflow config?